src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a `.scss` file. Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Please compile your SCSS file to a CSS file and update this link to point to the resulting CSS file.